ssl_cert_tracker/models.py

```diff
@@ -13,63 +13,93 @@
 :contact:    ali.rahmat@phsa.ca
 
 """
+from django.conf import settings
 from django.db import models
-from django.db.models import signals
-from django.dispatch import receiver
+from p_soc_auto_base.models import BaseModel
 
-class TestCertsData(models.Model):
-    """TestCertsData. Model struture for NMap result response. """
-    orion_id =  models.CharField(max_length=100, blank=True, null=True)
-    addresses =  models.CharField(max_length=100, blank=True, null=True)
+class NmapCertsData(BaseModel, models.Model):
+    """NmapCertsData. Model struture for NMap result response. """
+    orion_id = models.CharField(max_length=100, blank=True, null=True)
```

Comment:
the orion_id field needs to be replaced with a manytomany relationship that links to models in orion_integration